Documents/QA/TestPlan.md

```diff
@@ -0,0 +1,215 @@
+- [Test Plan](#test-plan)
+  - [Test Plan Approval](#test-plan-approval)
+  - [Table of Contents](#table-of-contents)
+  - [1. Introduction](#1-introduction)
+    - [a. Overview](#a-overview)
+    - [b. Targetted Audience](#b-targetted-audience)
+      - [Retro Gaming Player](#retro-gaming-player)
+      - [New Player](#new-player)
+      - [Speed Runner](#speed-runner)
+    - [c. Softwares/Hardwares](#c-softwares-hardwares)
+      - [FPGAs](#fpgas)
+      - [VERILOG](#verilog)
+  - [2. Test criteria](#2-test-criteria)
+    - [a. Suspension Criteria](#a-suspension-criteria)
+      - [Critical Bugs](#critical-bugs)
+    - [b. Entry Criteria](#b-entry-criteria)
+    - [c. Exit Criteria](#c-exit-criteria)
+  - [3. Test Strategy](#3-test-strategy)
+    - [a. Testing Scope](#a-testing-scope)
+    - [b. Testing Type](#b-testing-type)
+      - [Unit Tests](#unit-tests)
+      - [Functional Tests](#functional-tests)
+      - [Collaborative Game-Testing](#collaborative-game-testing)
+    - [c. Define Priorities](#c-define-priorities)
+    - [d. Test Tool](#d-test-tool)
+      - [System VERILOG](#system-verilog)
+      - [EDA Playground](#eda-playground)
+    - [e. Test Cases](#e-test-cases)
+    - [f. Test Reports](#f-test-reports)
+    - [g. Test Progression](#g-test-progression)
+  - [4. Schedule Estimation](#4-schedule-estimation)
+  - [5. Test Deliverables](#5-test-deliverables)
+  - [6. Glossary](#6-glossary)
+</details>
+
+## 1-Introduction
+### a-Overview
+We were tasked to recreate the game Frogger using an FPGA<sup><a href="#1">[1]</a></sup> using the verilog<sup><a href="#2">[2]</a></sup> language.
+
+### b-Targetted-Audience
+According to the types of players, our Frogger will have differents requierements.<br>
+<i>Take a look at personas definition in [functional specification](https://github.com/algosup/2024-2025-project-1-fpga-team-6/blob/main/Documents/FunctionalSpecifications/functionalSpecifications.md#4-personas-definition) for more informations</i>
+
+#### Retro-Gaming-Player
+What retro players need is a game that works just fine and that allows them to feels the sentation of old games.
+
+#### New-Player
+What new players need is a working game that let them experience the game without the risk of any issues coming from the game.
+
+#### Speed-Runner
+What speed runners need is a game with no bugs and fast enough to allow them to play fast without the game crashing.
+
+### c-Softwares-Hardwares
+For this frogger project, we have been imposed to use specifics hardware and software.
+
+#### FPGAs
+FPGA is an integrated circuit that can be repeatedly programmed.
+<br><i>Take a look at FPGA<sup><a href="#1">[1]</a></sup> in the glossary for more details.</i>
+
+#### VERILOG
+Verilog is the programming language we will use to program on FPGAs.
+<br><i>Take a look at verilog<sup><a href="#2">[2]</a></sup> in the glossary for more details.</i>
+
+## 2-Test-Criteria
+we use test criteria to know what the Quality Assurance will have to test.

```

Comment:
you begin your sentences with a Capital Letter, and here you do not.

Reply:
done